src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -115,4 +128,39 @@
         return commands;
     }
 
+    /**
+     * Creates the part of the gpg command, which specifies the proxy settings.
+     * @return String with proxy settings, e. g.
+     *  "http-proxy=http://someserver.com:8080"
+     * @throws IOException Thrown in case of XML parse error.
+     */
+    private String composeProxyClause() throws IOException {
+        String proxyClause = "proxy";
```

Comment:
I think qulice will fail on if you have variables/fields with camel case. It is better to name it just `clause`